dts/arm/st/stm32f4.dtsi

```diff
@@ -135,6 +135,29 @@
 			status = "disabled";
 			label = "SPI_1";
 		};
+
+		spi2: spi@40003800 {
```

Comment:
I wouldn't expect a spi node update in this PR related to USB?

Reply:
It was added with the last rebase. I will fix it.